utils/python/CIME/hist_utils.py

```diff
@@ -269,6 +284,10 @@ def generate_baseline(case, baseline_dir=None):
     if not os.path.isdir(basegen_dir):
         os.makedirs(basegen_dir)
 
+    if (os.path.isdir(os.path.join(basegen_dir,testcase)) and
+        case.get_value("MODEL") == "CESM"  and not allow_baseline_overwrite):
```

Comment:
My understanding from talking with @jgfouca was that allow_baseline_overwrite is now expected for the ACME workflow, too. Is that not true? I don't love the sprinkling of model-specific logic throughout the scripts, because this will start to get unmaintainable when we add other models, but if it's unavoidable, I can live with it for now.

However, isn't MODEL 'cesm' (lowercase), not 'CESM' (uppercase)?